src/psyclone/dynamo0p3.py

```diff
@@ -3933,11 +4300,7 @@ def check_args(call):
                 stencil_arg_count += 1
 
     # qr_argument
-    qr_required = False
-    for descriptor in call.ktype.func_descriptors:
-        if len(descriptor.operator_names) > 0:
-            qr_required = True
-    if qr_required:
+    if call.ktype.eval_shape in VALID_QUADRATURE_SHAPES:
```

Comment:
document args in parent check_args method docstring

Reply:
Done.